cupyx/jit/_compile.py

```diff
@@ -519,12 +547,21 @@ def _transpile_expr(expr, env):
         return Constant(expr.s)
 
     if isinstance(expr, ast.Subscript):
-        # # TODO(asi1024): Fix.
-        # value = _transpile_expr(expr.value, env)
-        # if isinstance(expr.slice, ast.Index):
-        #     index = _transpile_expr(expr.slice.value, env)
-        #     return value + '[' + index + ']'
-        raise NotImplementedError('Not implemented.')
+        value = _transpile_expr(expr.value, env)
+        index = _transpile_expr(expr.slice, env)
+        if is_constants([value, index]):
+            return Constant(value[index])
+        value = _to_cuda_object(value, env)
+        index = _to_cuda_object(index, env)
+
+        if not isinstance(value.ctype, _types.Array):
+            raise ValueError(f'{value.code} must be Array type.')
+        if value.ctype.ndim != 1:
+            raise NotImplementedError('Not implemented for ndim > 1.')
```

Comment:
Currently, only 1-dim arrays are supported, but we are planning to support also multi-dimension arrays in the next PR.